pos-print/build.gradle

```diff
@@ -86,6 +86,8 @@ dependencies {
   compile 'org.slf4j:slf4j-api:1.6.3'
   compile 'ch.qos.logback:logback-classic:0.9.30'
 
+  compile group: 'com.fasterxml.jackson.core', name: 'jackson-core', version: '2.8.0'
```

Comment:
dependencies should follow the same pattern as existing ones. 

compile 'ch.qos.logback:logback-classic:0.9.30'